tests/testthat/test-values.R

```diff
@@ -270,16 +294,12 @@ test_that('sequences - logical', {
 
 
 test_that('sampling - character and logical', {
-  set.seed(9950)
+  #set.seed(9950)
   expect_equal(
     sort(unique(value_sample(surv_dist(), 500))), sort(surv_dist()$values)
   )
-  set.seed(9950)
+  #set.seed(9950)
```

Comment:
If we are somewhat confident we don't need this, I would just remove it entirely